CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+
+- Once you submit your PR, we will review it. But, the first thing you're going to want to do is a [self review](#self-review).
+- After that, we may have questions, check back on your PR to keep up with the conversation.
+
+### Your pull request is merged!
+
+Congratulations! The whole MicroStrategy community thanks you. :sparkles:
+
+## Contribution Guide
+
+As you make changes keep the following in mind: 
+1. If the api is not available publicly, we will not add the api workflow in playground.
+2. All the workflow in playground should be ready to use in demo or brand new cloud environment without server side setup.
+3. Exception to (2), if the workflow requires some configuration, we should clearly document the requirement in documentation.
+
+### Variables
+
+There are many different types of [variables](https://learning.postman.com/docs/sending-requests/variables/) available in postman. In our playground, we use environment variables and local variables.
+
+#### Environment Variables
+
+Environment variables are scoped to the [environment](https://learning.postman.com/docs/sending-requests/managing-environments/) you selected. We provide two environments in our playground. One is the [demo environment](https://www.postman.com/microstrategysdk/workspace/microstrategy-rest-api/environment/16131298-aac3a1ea-b878-488a-bf80-f4dc5f50a831) environment and the other is a [template](https://www.postman.com/microstrategysdk/workspace/microstrategy-rest-api/environment/16131298-531df55b-93e4-4911-a93d-6d853455e9c5) for a new cloud environment.
+
+In these environments, we provide some long-lived environment variables such as:
+- baseUrl
+- projectId
+- username
+- password
+- loginMode
+Please use these long-lived environment variables over creating your own, whenever possible.
+Note: In the template environment, please change these values to suit your environment.
+
+Environment variables are used to store values that may be used across more than one apis or workflows. For PRs that add/modify content from our collections, you can create/edit these variables through [pre-request scripts and tests](https://learning.postman.com/docs/sending-requests/managing-environments/#setting-environment-variables-from-scripts).
+
+For environment variable naming convention and usage:
+1. We adopt camelCase for variable names.
+  An example can be "`dossierId`" instead of "`DossierID`".
+2. We enforce a "rd_" prefix to runtime generated environment variables(set inside test scripts or pre-request scripts).
+  An example can be "rd_instanceId". This variable is set in the test script of an instance creation api.
+3. We do not have a "rd_" prefix to long-lived environment variables nor using non-runtime environment variables.
+  An example can be "`dossierId`" instead of "`rd_dossierId`" because we plan to use the provided "`dossierId`" in the environment rather than the runtime generated one.
+4. There are special rules that apply to each collection
```

Comment:
```suggestion
- We adopt camelCase for variable names.
  An example can be "`dossierId`" instead of "`DossierID`".
- We enforce a "rd_" prefix to runtime generated environment variables(set inside test scripts or pre-request scripts).
  An example can be "rd_instanceId". This variable is set in the test script of an instance creation api.
- We do not have a "rd_" prefix to long-lived environment variables nor using non-runtime environment variables.
  An example can be "`dossierId`" instead of "`rd_dossierId`" because we plan to use the provided "`dossierId`" in the environment rather than the runtime generated one.
- There are special rules that apply to each collection
```